src/pages/homePage/HomePage.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchDetails } from '../../redux/features/detailsSlice';
+import CurrencyHome from '../../components/currencyCard/CurrencyCard';
+import Navbar from '../../components/navbar/Navbar';
+import './HomePage.css';
+
+const HomePage = () => {
+  const dispatch = useDispatch();
+  const { currencyList, status } = useSelector((store) => store.details);
+
+  useEffect(() => {
+    if (status === 'idle') {
+      dispatch(fetchDetails());
+    }
+  }, [status, dispatch]);
+
+  return (
+    <div className="home-page">
+      <Navbar type="home" />
+      <div className="details-home-container">
+        {currencyList.map((currency) => (
+          <NavLink key={currency.id} to={`/${currency.id}`} className="nav-link">
+            <CurrencyHome {...currency} />
+          </NavLink>
+        ))}
+      </div>
```

Comment:
- [ ]   Your Home page should have away of filtering Cryptometrics  referring to something like category name as it is suggested in in requirement etc.
![image](https://user-images.githubusercontent.com/43877689/225978123-6de5474b-86d2-4c3d-b572-3b4f795edfd5.png)